### PR TITLE
Add SC 5.0.1

### PR DESCRIPTION
**Pull request by alexh-sauce**

### CODE REVIEW

**Comment by alexh-sauce** [comment]
@Choraden Please see f6264ee9, it looks like the 5.0.0 tag was listed twice. Was it meant to be what I changed docker.md to?

**Comment by aluedeke** [comment]
@alexh-sauce did none of the changes mentioned in the changelog require a change/addition in docs?

File: docs/secure-connections/sauce-connect-5/installation/docker.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ Here are some benefits/use cases for using containerized Sauce Connect Proxy:
    ```
    - To use a specific version, add it as a tag:
    ```bash
-   $ docker pull saucelabs/sauce-connect:5.0.0-amd64
+   $ docker pull saucelabs/sauce-connect:5.0.1-amd64
    ```
     <details><summary>Supported tags</summary>
-      - 5, 5.0, 5.0.0, 5.0.0-arm64v8<br/>
+      - 5, 5.0, 5.0.1, 5.0.1-arm64v8<br/>
```
[the remark this answers is not in the document]
Comment (alexh-sauce) [reply docs/secure-connections/sauce-connect-5/installation/docker.md]: Fixed in ac5c5c47

**Comment by alexh-sauce** [comment]
> @alexh-sauce did none of the changes mentioned in the changelog require a change/addition in docs?

I believe they're all taken care of in the automated https://github.com/saucelabs/sauce-docs/pull/2684 PR

**Review by Choraden — APPROVED** [review]

**Pull request merged**